atmo/clusters/provisioners.py

```diff
@@ -0,0 +1,84 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+from ..provisioners import Provisioner
+
+
+class ClusterProvisioner(Provisioner):
+    log_dir = 'clusters'
+
+    def start(self, user_email, identifier, size, emr_release,
+              public_key):
+        """
+        Given a user's email, a cluster identifier, a worker count, and a user
+        public key, spawns a cluster with the desired properties and returns
+        the jobflow ID.
```

Comment:
The comment doesn't match the args but I think it might be better if they didn't anyway. Maybe remove them in the comment? Easier to maintain.